apps/web/src/config/constants/endpoints.ts

```diff
@@ -38,7 +38,7 @@ export const ACCESS_RISK_API = 'https://red.alert.pancakeswap.com/red-api'
 export const CELER_API = 'https://api.celerscan.com/scan'
 
 export const INFO_CLIENT_WITH_CHAIN = {
-  [ChainId.BSC]: INFO_NR_CLIENT,
+  [ChainId.BSC]: 'https://pancakeswap.nodereal.io/graphql/',
```

Comment:
Should we also change the `INFO_CLIENT_WITH_CHAIN` ?
`apis/farms/src/lpAprs.ts`